### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by z3roshot**
I'm not certain if the intent for this message is to create a fork under your own github account or create a branch in the ATS repo and create the PR from their.

I believe the former to be correct, and if so I propose this word change.

### CODE REVIEW

**Comment by natoverse** [comment]
I could go either way on this myself - either (a) fork the repo personally, make changes, and PR, or (b) make a branch in the main repo, make changes, and PR. We delete branches as soon as the PR is merged, so I'm not concerned about accumulation if (b) is used.

Anyway, either way the wording should be more clear because "branch the repo" sounds like forking. But which mechanism is used I'd recommend leaving up to individual devs based on personal preference. Maybe reword to "fork the repo or create a branch"?

**Comment by z3roshot** [comment]
That works for me @natoverse!

**Comment by z3roshot** [comment]
Interesting, the Travis build failed during "gulp deploy".

Anyway I'll close this for now.

**Comment by natoverse** [comment]
Hmm, looks like the auth failed. I thought the travis config should only try to run the gulp deploy on master commits anyway, so it shouldn't even have tried. I'll update the wording on master and see if it fails again there.
